src/coldfront_plugin_api/scim_v2/users.py

```diff
@@ -38,7 +38,7 @@ class ListUsers(APIView):
     * Only admin users are able to access this view.
     """
     authentication_classes = auth.AUTHENTICATION_CLASSES
-    permission_classes = [IsAdminUser]
+    permission_classes = [IsAdminUser] 
```

Comment:
You have added some whitespace here by mistake, your Pull Requests shouldn't introduce unrelated changes like this.